internal/nodes/avi_model_nodes.go

```diff
@@ -915,6 +915,7 @@ type AviVSVIPNode struct {
 	SubnetIP                string
 	SubnetPrefix            int32
 	NetworkNames            []string
+	EnablePublicIP          *bool
```

Comment:
this will need to be part of the checksum calculation. Also could you please add this field's checks in the existing AviInfraSetting UTs too.

Reply:
updated